src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -5,28 +5,331 @@
+        <p>{{ fieldClpResourcesIntroText }}</p>
+
+        <!-- List of resources -->
+        {% if fieldClpResources != empty %}
+          <div class="resources-list">
+            {% for resource in fieldClpResources %}
+              <div class="resource">
+                <h3>{{ resource.entity.name }}</h3>
+                <p>{{ resource.entity.fieldDescription }}</p>
+                <a href="{{ resource.entity.fieldMediaExternalFile.url }}" rel="noreferrer noopener" target="_blank">Download (PDF)</a>
+              </div>
+            {% endfor %}
+          </div>
+        {% endif %}
+
+        <!-- Call to action -->
+        {% if fieldClpResourcesCta %}
+          <a class="usa-button-secondary" href="{{ fieldClpResourcesCta.entity.fieldButtonLink.url }}" rel="noreferrer noopener" target="_blank">
+            {{ fieldClpResourcesCta.entity.fieldButtonLabel }}
+          </a>
+        {% endif %}
+      </div>
+
+      <!-- Events -->
+      <div class="events">
+        <!-- Title -->
+        <p class="grey-header">EVENTS</p>
+        <h2>{{ fieldClpEventsHeader }}</h2>
+
+        <!-- List of events -->
+        {% if fieldClpEventsReferences != empty %}
+          <div class="events-list">
+            {% for eventReference in fieldClpEventsReferences %}
+              <div class="event-reference">
+                <!-- Title -->
+                <h3>
+                  <a href="{{ eventReference.entity.fieldLink.url }}" rel="noreferrer noopener" target="_blank">
+                    {{ eventReference.entity.fieldListing.entity.fieldDescription }}
+                  </a>
+                </h3>
+
+                <!-- Description -->
+                {{ eventReference.entity.fieldBody.processed }}
+
+                <!-- When -->
+                <div class="event-reference-data-point">
+                  <p><strong>When</strong></p>
+                  <p>
+                    {% if eventReference.entity.fieldDatetimeRangeTimezone.value != empty %}
+                      {{eventReference.entity.fieldDatetimeRangeTimezone.value | dateFromUnix: "dddd, MMM D, h:mm A" }} -
+                      <br />
+                      {{eventReference.entity.fieldDatetimeRangeTimezone.endValue | dateFromUnix: "dddd, MMM D, h:mm A" }}
+                    {% endif %}
+                    {% if eventReference.entity.fieldDatetimeRangeTimezone.timezone != empty %}
+                      {{ eventReference.entity.fieldDatetimeRangeTimezone.timezone | timezoneAbbrev: fieldDatetimeRangeTimezone.value }}
+                    {% endif %}
+                  </p>
+                </div>
+
+                <!-- Where -->
+                <!-- NOT SURE WHAT TO PUT HERE: https://dsva.slack.com/archives/C01DS1XDEQ0/p1611332573004800?thread_ts=1611331917.004100&cid=C01DS1XDEQ0 -->
+                <div class="event-reference-data-point">
+                  <p><strong>Where</strong></p>
+                  <a href="" rel="noreferrer noopener" target="_blank">
```

Comment:
Is this `href` intentionally blank because you're not sure what to put in this section?

Reply:
Correct 🙂

Reply:
Again this is all just boilerplate